docker/pdo_services.dockerfile

```diff
@@ -55,12 +55,9 @@ EXPOSE 7001 7002 7003 7004 7005
 EXPOSE 7101 7102 7103 7104 7105
 EXPOSE 7201 7202 7203 7204 7205
 
-ARG PDO_HOSTNAME
+ARG PDO_HOSTNAME=localhost
 ENV PDO_HOSTNAME=$PDO_HOSTNAME
 
-ARG PDO_LEDGER_URL
-ENV PDO_LEDGER_URL=$PDO_LEDGER_URL
```

Comment:
(same comment as above)

Reply:
as i mentioned above... the only good reason for having these in the dockerfile is that they simplify construction of pre-configured images. but... i'm with you... pdo-hostname and pdo-ledger-url should NOT be build variables. we should pre-configure for localhost and actually set the values we want in the start_* scripts.

Reply:
if you are keeping hostname then you have to keep the ledger_url. there is no reason whatsoever to have one without the other. either drop both or keep both.